metaci/release/models.py

```diff
@@ -79,6 +102,12 @@ def get_production_date():
     change_case_link = models.URLField(
         _("change case link"), max_length=1024, null=True, blank=True
     )
+    sandbox_implemention_step = models.ForeignKey(
+        "release.ImplementationStep",
+        on_delete=models.SET_NULL,
+        null=True,
+        related_name="releases",
+    )
```

Comment:
It doesn't make sense to have a release field on ImplementationStep _and_ implementation step fields on Release. One direction is enough to create a relationship; 2 directions will be difficult to keep in sync.

I don't think we need the field on Release, because we can tell which ImplementationStep is the sandbox push one based on which Plan it's related to.